Lib/fontTools/varLib/__init__.py

```diff
@@ -601,6 +601,24 @@ def _add_MVAR(font, masterModel, master_ttfs, axisTags):
 		mvar.ValueRecord = sorted(records, key=lambda r: r.ValueTag)
 
 
+def _add_BASE(font, masterModel, master_ttfs, axisTags):
+
+	log.info("Generating BASE")
+
+	merger = VariationMerger(masterModel, axisTags, font)
+	merger.mergeTables(font, master_ttfs, ['BASE'])
+	store = merger.store_builder.finish()
+
+	if not store.VarData:
```

Comment:
i think it's preferable to do this to check for an empty item variation store:
```suggestion
	if not store.VarRegionList.Region:
```

there could be empty items in VarData array, but with no regions they are ineffective so it is useless and can be dropped

Reply:
anyway, I see _merge_OTL also does what you did, I'm ok to keep as is